presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -600,30 +600,30 @@ public void testStructTimestamps(StorageFormat format)
                             format("row(col %s)", type),
                             format("array(map(%1$s, row(col array(%1$s))))", type)));
 
-            // Check the values as varchar
-            String query = format(
-                    "SELECT"
-                            + "   id,"
-                            + "   CAST(arr[1] AS VARCHAR),"
-                            + "   CAST(map_entries(map)[1][1] AS VARCHAR)," // key
-                            + "   CAST(map_entries(map)[1][2] AS VARCHAR)," // value
-                            + "   CAST(row.col AS VARCHAR),"
-                            + "   CAST(map_entries(nested[1])[1][1] AS VARCHAR)," // key
-                            + "   CAST(map_entries(nested[1])[1][2].col[1] AS VARCHAR)" // value
-                            + " FROM %s"
-                            + " ORDER BY id",
-                    tableName);
-            System.out.println(query);
-            onPresto().executeQuery(query);
-            assertThat(onPresto()
-                    .executeQuery(query))
-                    .as("timestamp containers on %s", format.getName().toLowerCase(Locale.ENGLISH))
-                    .containsExactly(TIMESTAMPS_FROM_HIVE.stream()
-                            .sorted(comparingInt(TimestampAndPrecision::getId))
-                            .map(e -> new Row(Lists.asList(
-                                    e.getId(),
-                                    nCopies(6, e.getReadValue(precision)).toArray())))
-                            .collect(toList()));
+            // // Check the values as varchar
+            // String query = format(
+            //         "SELECT"
+            //                 + "   id,"
+            //                 + "   CAST(arr[1] AS VARCHAR),"
```

Comment:
Let's add `// TODO https://github.com/prestosql/presto/issues/6350`

As a temporary workaround, let's use cast to JSON:

```
presto:default> SELECT CAST(col AS json) FROM test;
              _col0
---------------------------------
 ["2020-12-22 12:19:45.7786160"]
```

and then
```
presto:default> SELECT regexp_replace(json_format(CAST(col AS json)), '^\["|"\]$') FROM test;
            _col0
-----------------------------
 2020-12-22 12:19:45.7786160
```